developer-docs-site/docs/tutorials/validator-node/intro.md

```diff
@@ -88,3 +88,9 @@ Follow the links below to begin your installation:
 * [Using AWS](using-aws.md)
 * [Using Docker](using-docker.md)
 * [Using Aptos source code](using-source-code.md)
+
+### Troubleshooting
+
+**Q: Do we need to run validator node and FullNode on two different machines, in order to get two different IPs?**
+
+**A:** We recommend that you run the validator node and the FullNode on two separate and independent machines that are well-provisioned and isolated from each other. Also, the FullNode IP is optional and you can leave it as blank. 
```

Comment:
We should just put this up in the tip section above... or make a note section there as well

also the recommendation is that the validator and fullnode should be run in a way that guarantees resource isolation. The rest of this doesn't seem to be clear.